course/meetings/eeb-marty-2017-04-13.md

```diff
@@ -0,0 +1,59 @@
+---
+contact: Asher Cutter, Martin Krkosek
+department: Ecology and Evolutionary Biology
+uoftcoders: Madeleine, Luke, Joel, Elliott
+date: 2017-04-13
+---
+
+# Action Items
+- Madeleine: Add some specifics to the ecology topics in the syllabus
```

Comment:
No stress! I got lab meeting on Tuesday so will probably not have time to look at until after that.